### PR TITLE
Add icon capability to api designer

### PR DESCRIPTION
**Pull request by DimalChandrasiri**

### CODE REVIEW

File: tool-plugins/vscode/resources/images/icons/apidesign.svg [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ ~   Unless required by applicable law or agreed to in writing, software
+ ~   distributed under the License is distributed on an "AS IS" BASIS,
+ ~   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~   See the License for the specific language governing permissions and
+ ~   limitations under the License.
+-->
+
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 16 16" style="enable-background:new 0 0 16 16;" xml:space="preserve">
+<path d="M9.4000244,5.0899658H10v3.0100098H9.4000244V5.0899658z M2.4400024,6.5999756
```
Comment (jeradrutnam) [inline tool-plugins/vscode/resources/images/icons/apidesign.svg]: Make the icon color: #424242

File: tool-plugins/vscode/resources/images/icons/apidesign-inverse.svg [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ ~   distributed under the License is distributed on an "AS IS" BASIS,
+ ~   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~   See the License for the specific language governing permissions and
+ ~   limitations under the License.
+-->
+
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 16 16" style="enable-background:new 0 0 16 16;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#C5C5C4;}
```
Comment (jeradrutnam) [inline tool-plugins/vscode/resources/images/icons/apidesign-inverse.svg]: Make the fill color inline in the path

Comment (DimalChandrasiri) [reply tool-plugins/vscode/resources/images/icons/apidesign-inverse.svg]: fixed with commit 403346a

File: tool-plugins/vscode/resources/images/icons/apidesign.svg (hunk 1 above)
Comment (DimalChandrasiri) [reply tool-plugins/vscode/resources/images/icons/apidesign.svg]: fixed with commit 403346a

**Pull request merged**